src/main.js

```diff
@@ -195,6 +195,21 @@ module.exports = {
     this.subscriptions.dispose()
   },
 
+  async restartESLintWorker() {
+    return new Promise((resolve) => {
+      if (this.worker !== null) {
+        this.worker.terminate()
+        this.worker = null
+      }
+      const initializeESLintWorker = () => {
+        this.worker = new Task(require.resolve('./worker.js'))
+        resolve()
+      }
+      // Initialize the worker during an idle time
+      window.requestIdleCallback(initializeESLintWorker)
```

Comment:
This should be added to the `idelCallbacks` `Set` (The call up on L185 also should be added). Otherwise we risk initializing a worker when the package is deactivated.